pkg/claude/agentd/dashboard.css

```
@@ -0,0 +1,1167 @@
+  background: #161b22; border: 1px solid #2a3340; border-radius: 6px;
+  padding: 10px 14px; margin-bottom: 8px;
+}
+.msg-card.msg-unread { border-left: 3px solid #58a6ff; }
+.msg-head { display: flex; align-items: center; gap: 6px; font-size: 13px; }
+.msg-dot { color: #58a6ff; font-size: 10px; }
+.msg-from { font-weight: 600; color: #d4d4d4; }
+.msg-group { color: #7d8590; }
+.msg-id { color: #7d8590; font-family: ui-monospace, monospace; font-size: 11px; }
+.msg-time { color: #7d8590; font-size: 12px; }
+.msg-subject { font-weight: 600; color: #d4d4d4; margin-top: 6px; }
+.msg-body { color: #c9d1d9; margin-top: 4px; white-space: pre-wrap; word-break: break-word; }
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)